addons/binding/org.openhab.binding.rfxcom/README.md

```diff
@@ -104,6 +105,8 @@ This binding currently supports following channels:
 | motion | Switch | Motion detection sensor state. |
 | rainrate | Number | Rain fall rate in millimeters per hour. |
 | raintotal | Number | Total rain in millimeters. |
+| rawmessage | String | Hexadecimal string of the raw RF message. |
```

Comment:
Yes, you have to turn them on in the RFXCOM config, but that's the same for any device, it has to be enabled in the config. I don't think it needs expicitly calling out, although it might not be a bad idea to make rawmessage and rawpayload work for all messages, regardless of type.